packages/fx-core/package.json

```diff
@@ -61,8 +61,8 @@
     "@azure/arm-sql": "^9.0.0",
     "@azure/arm-storage": "^17.2.1",
     "@azure/arm-subscriptions": "^5.0.0",
-    "@azure/identity": "^1.5.2",
-    "@azure/msal-node": "^1.0.0",
+    "@azure/identity": "^3.1.3",
```

Comment:
@xiaolang124 any specific test case needs to be added for such major version change?

Reply:
I think if e2e test cases work fine, this change is fine.

Reply:
@xiaolang124 I've triggered the E2E test pipeline: https://github.com/OfficeDev/TeamsFx/actions/runs/4050434854
There are still several failures, it seems that they are not related to this change.

Reply:
Latest E2E: https://github.com/OfficeDev/TeamsFx/actions/runs/4060696750